### DISCUSSION TITLE
How to ignore an app that doesn't work with its name and doesn't have a specific pasteboard? (input-leap, barrier, synergy etc.)

### DISCUSSION THREAD

**Discussion by taylankasap** [p0deje/Maccy#956] [Q&A]
input-leap is an app that let's you control multiple computers with single keyboard/mouse. So it also syncs clipboard. But I don't want my personal computer's clipboard to get saved in Maccy. I'm trying to ignore clipboard values that come from input-leap app.

This scenario is similar to Universal Clipboard that is mentioned in the FAQ. But pasteboard viewer does not show anything specific to input-leap. Selecting InputLeap.app also does not ignore it.

![image](https://github.com/user-attachments/assets/b3e23d3f-f12b-49cb-b7f3-6c9fbcc08135)

On macos, whatever is on the foreground looks like the source. In this case, "Google Chrome".

Any ideas on how to proceed with this?

**Comment by p0deje** [comment]
Unfortunately, I don't know of any other way to ignore the clipboard for such apps.
